### PR TITLE
Remove unused dart:async imports

### PR DESCRIPTION
**Pull request by MichaelRFairhurst**
Since Dart 2.1, Future and Stream have been exported by dart:core

### CODE REVIEW

File: packages/devtools_app/lib/src/memory/memory_heap_tree_view.dart [hunk 1]
```diff
@@ -347,9 +346,7 @@ class HeapTreeViewState extends State<HeapTree>
                     ? 'Graphing...'
                     : _isSnapshotGrouping
                         ? 'Grouping...'
-                        : _isSnapshotComplete
-                            ? 'Done'
-                            : '...',
+                        : _isSnapshotComplete ? 'Done' : '...',
```
Comment (kenzieschmoll) [inline packages/devtools_app/lib/src/memory/memory_heap_tree_view.dart:349]: you'll need to revert this change to get the bots to pass - the failing bot is formatting based on Flutter 1.22.0-12.0.pre

**Review by kenzieschmoll — APPROVED** [review]

File: packages/devtools_app/lib/src/memory/memory_heap_tree_view.dart (hunk 1 above)
Comment (MichaelRFairhurst) [reply packages/devtools_app/lib/src/memory/memory_heap_tree_view.dart]: Done! My bad, auto-save formatting :)

**Comment by MichaelRFairhurst** [comment]
Looks like I don't have approval to merge!

**Comment by MichaelRFairhurst** [comment]
Thanks!

**Pull request merged**